libs/langchain-google-genai/src/tests/context_caching.int.test.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-process-env */
+
+import { test } from "@jest/globals";
+
+import { fileURLToPath } from "node:url";
+import * as path from "node:path";
+
+import { FileState, UploadFileResponse } from "@google/generative-ai/server";
+import { GoogleGenerativeAIContextCache } from "../context_caching.js";
+import { ChatGoogleGenerativeAI } from "../chat_models.js";
+
+const model = new ChatGoogleGenerativeAI({});
+let fileResult: UploadFileResponse;
+
+beforeAll(async () => {
+  // Download video file and save in src/tests/data
+  // curl -O https://storage.googleapis.com/generativeai-downloads/data/Sherlock_Jr_FullMovie.mp4
+  const displayName = "Sherlock Jr. video";
+
+  const filename = fileURLToPath(import.meta.url);
+  const dirname = path.dirname(filename);
+  const pathToVideoFile = path.join(dirname, "/data/Sherlock_Jr_FullMovie.mp4");
```

Comment:
Can we use a smaller file for this test? Maybe one of the files that already exist in the repo?